modules/502-delivery/openapi/openapi-case-tests.yaml

```diff
@@ -0,0 +1,7 @@
+positive:
+  values:
+    - auth:
+        webui:
```

Comment:
```suggestion
        webui:
```
Looks like a copy paste from the upmeter module and IDK how it works.

Reply:
Removed, test cases fulfilled